crates/aptos/src/common/types.rs

```diff
@@ -742,8 +742,8 @@ pub trait ParsePrivateKey {
 pub struct HardwareWalletOptions {
     /// Derivation Path of your account in hardware wallet
     ///
-    /// e.g format - [m/44'/637'/0'/0'/0]
-    /// Make sure your wallet is unblocked and have Aptos opened
+    /// e.g format - m/44\'/637\'/0\'/0\'/0\'
+    /// Make sure your wallet is unlocked and have Aptos opened
```

Comment:
Technically, it is the original one, but then you have to pass it in through the CLI, which gets messy.

Reply:
Yeah. I think people will forget to escape the `'`.  Hopefully this make it clear when using the CLI